src/phantom.js

```diff
@@ -45,7 +36,36 @@ export default class Phantom {
         }
 
         let pathToShim = path.normalize(__dirname + '/shim.js');
-        logger.debug(`Starting ${phantomPath} ${args.concat([pathToShim]).join(' ')}`);
+
+        let loggerLevel = process.env.DEBUG === 'true' ? 'debug' : 'info';
+        if(typeof config.logger === 'object') {
```

Comment:
I turned the problem around, but if we put a default value for `config` inside `index.js#create`, we will have to check its value, and these values (like the logLevel) or the value of will probably have to be checked again in the `phantom` constructor, so... It may not help. As I understand it, it would be something like : 
```js
import Phantom from "./phantom";
import winston from "winston";

const logLevel = process.env.DEBUG === 'true' ? 'debug' : 'info';
const logger = new winston.Logger({
    transports: [
        new winston.transports.Console({
            level: logLevel,
            colorize: true
        })
    ]
});

/**
 * Retuns a new instance of Phantom class
 * @param args
 * @returns {Promise}
 */
module.exports.create = (args, config = {logger: logger, logLevel: logLevel}) => {
    if (config.logger === logger) {
        logger.transports.console.level = logLevel;
    }
    return new Promise(resolve => resolve(new Phantom(args, config)));
};
```